modules/api_docs/jsx/swagger-ui_custom.js

```diff
@@ -3,6 +3,7 @@ import 'swagger-ui-dist/swagger-ui.css';
 
 window.addEventListener('load', () => {
   const ui = new SwaggerUIBundle({
+    // eslint-disable-next-line no-undef
```

Comment:
why is this here?

Reply:
Hi @driusan the variable "schema_urls" is undefined and created in php. So the eslint-disable-next-line no-def is one solution. @laemtl do you know if there's a more elegant solution for this?

Reply:
I still don't understand.. how is php defining things in the javascript file? Is it a global variable? If so, can it be explicitly defined as one somewhere?

Reply:
@driusan I meant the php line `$this->tpl_data['schema_urls'] = json_encode($api_specs);`. I think the global variable solution is best as well. I added the change in latest commit.